Content.Server/Administration/Systems/SuperBonkSystem.cs

```diff
@@ -0,0 +1,100 @@
+using Content.Server.Administration.Components;
+using Content.Shared.Climbing.Components;
+using Content.Shared.Climbing.Events;
+using Content.Shared.Climbing.Systems;
+using Content.Shared.Interaction.Components;
+using Content.Shared.Mobs;
+using Content.Shared.Mobs.Components;
+
+namespace Content.Server.Administration.Systems;
+
+public sealed class SuperBonkSystem: EntitySystem
+{
+    [Dependency] private readonly SharedTransformSystem _transformSystem = default!;
+    [Dependency] private readonly BonkSystem _bonkSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<SuperBonkComponent, ComponentShutdown>(OnBonkShutdown);
+    }
+
+    public void StartSuperBonk(EntityUid target, float delay = 0.1f, bool stopWhenDead = false )
+    {
+
+        var hadClumsy = EnsureComp<ClumsyComponent>(target, out _);
```

Comment:
pretty sure you can just use the EnsureComp overload without the out var.

Reply:
Not sure which one you are referring to. The only EnsureComp overload I can find which returns a bool and doesn't take an out parameter needs an Entity as an input.